code/modules/projectiles/guns/pistols.dm

```diff
@@ -56,7 +56,7 @@
 	icon_state = "tp14"
 	item_state = "tp14"
 	caliber = CALIBER_9X19 //codex
-	max_shells = 21 //codex
+	max_shells = 14 //codex
```

Comment:
this kills the tp14 niche. All pistols should stay unique

Reply:
Okay. I'll remove it and we'll see what the sunder change does.